source/common/upstream/health_checker_impl.cc

```diff
@@ -256,13 +258,24 @@ void HttpHealthCheckerImpl::HttpActiveHealthCheckSession::decodeHeaders(
   }
 }
 
+void HttpHealthCheckerImpl::HttpActiveHealthCheckSession::decodeData(Buffer::Instance& data,
+                                                                     bool end_stream) {
+  if (!parent_.expected_response_.empty() && response_body_->length() < 1024) {
```

Comment:
Sorry, missed this one. Please define a named constant for `1024`

Reply:
Thank you! Done.